tendrl/commons/message.py

```diff
@@ -153,7 +153,7 @@ def serialize_message(obj):
         serial = obj.isoformat()
         return serial
     elif isinstance(obj, Exception):
-        return yaml.safe_dump(obj)
+        return yaml.safe_dump(obj.message)
```

Comment:
I am changing it to dump of exception object itself. But  @r0h4n @shtripat  @nthomas-redhat  ^^ What do you suggest